### PR TITLE
ToggleExtension: support running on all wikis at once

### PR DESCRIPTION
**Pull request by Universal-Omega**

### CODE REVIEW

File: maintenance/toggleExtension.php [hunk 1]
```diff
@@ -9,30 +9,66 @@
...
+use MediaWiki\WikiMap\WikiMap;
 use Miraheze\ManageWiki\Helpers\ManageWikiExtensions;
 
 class ToggleExtension extends Maintenance {
 	public function __construct() {
 		parent::__construct();
 		$this->addArg( 'ext', 'The ManageWiki name of the extension.', true );
 		$this->addOption( 'disable', 'Disable the extension. If not given, enabling is assumed.' );
+		$this->addOption( 'all-wikis', 'Run on all wikis present in $wgLocalDatabases.' );
+		$this->addOption( 'no-list', 'Only list wikis after it has ran on them. This may speed up execution.' );
```
Comment (Universal-Omega) [inline maintenance/toggleExtension.php:21]: Whoops, I guess this is kinda an inaccurate description.

Comment (redbluegreenhat) [reply maintenance/toggleExtension.php:21]:
```suggestion
		$this->addOption( 'quiet', 'Suppress output' );
```

could be a better name and description.

Comment (Universal-Omega) [reply maintenance/toggleExtension.php:21]: I thought about that but it doesn't really suppress output just doesn't list each one, it still has output at the end.

Comment (redbluegreenhat) [reply maintenance/toggleExtension.php:21]: Hmm, how about this then?

```suggestion
		$this->addOption( 'no-list', "Don't list on which wikis this script has ran." );
```

**Comment by paladox** [comment]
Could add a --confirm flag instead of relying on user interaction with the prompt.

File: maintenance/toggleExtension.php (hunk 1 above)
Comment (redbluegreenhat) [reply maintenance/toggleExtension.php:21]: actually, not even I like that last suggestion.

**Comment by Universal-Omega** [comment]
> Could add a --confirm flag instead of relying on user interaction with the prompt.

That's not a bad idea, I like that.

**Review by redbluegreenhat — APPROVED** [review]
LGTM

**Pull request merged**